app/models/id_card_attributes.rb

```diff
@@ -16,10 +16,10 @@ def traits
       'edipi' => @user.edipi,
       'firstname' => @user.first_name,
       'lastname' => @user.last_name,
-      'address' => @user.va_profile&.address&.street || '',
-      'city' => @user.va_profile&.address&.city || '',
-      'state' => @user.va_profile&.address&.state || '',
-      'zip' => @user.va_profile&.address&.postal_code || '',
+      'address' => @user&.address&.street || '',
```

Comment:
I think we don't need the first safety operator for these (so `@user.address&.street`), since we know that `@user` won't be nil